src/lib/ndpi_content_match.c.inc

```diff
@@ -717,6 +717,31 @@ static ndpi_network host_protocol_list[] = {
   { 0x3353EED8 /* 51.83.238.216/30 */, 30, NDPI_PROTOCOL_ANYDESK },
   { 0x3353EEDC /* 51.83.238.220/31 */, 31, NDPI_PROTOCOL_ANYDESK },
 
+  /*
+   * Edgecast CDN
+   * Uses captcha to prevent web scraping.
+   * https://support.globaldots.com/hc/en-us/articles/115004003849-Edgecast-IP-addresses
+   */
```

Comment:
Reading that page, it seems to me these addresses are used only from Edgecast servers to connect to the origin.
In other words, if I am right, a client connecting to a site served by this CDN does not connect to one of this addresses...

Probably the real list we want is https://bgp.he.net/AS15133#_prefixes (and that seems confirmed by https://developer.edgecast.com/cdn/api/Content/Media_Management/Get-CDN-IP-Blocks.htm)

Anyway, this list might be useful anyway.

Reply:
I have only skimmed that page, unfortunately. You're right, AS15133 seems promising.